README.md

```diff
@@ -23,45 +20,115 @@ Checkout the branch.
+|`persistence.persistentVolumeClaim.database.existingClaim`     | Use the existing PVC which must be created manually before bound | |
+|`persistence.persistentVolumeClaim.database.storageClass`     | Specify the `storageClass` used to provision the volume. Or the default StorageClass will be used(the default). Set it to `-` to disable dynamic provisioning | |
+|`persistence.persistentVolumeClaim.database.subPath`     | The sub path used in the volume | `database` |
+|`persistence.persistentVolumeClaim.database.accessMode`     | The access mode of the volume | `ReadWriteOnce` |
+|`persistence.persistentVolumeClaim.database.size`     | The size of the volume | `1Gi` |
+|`persistence.persistentVolumeClaim.redis.existingClaim`     | Use the existing PVC which must be created manually before bound | |
+|`persistence.persistentVolumeClaim.redis.storageClass`     | Specify the `storageClass` used to provision the volume. Or the default StorageClass will be used(the default). Set it to `-` to disable dynamic provisioning | |
+|`persistence.persistentVolumeClaim.redis.subPath`     | The sub path used in the volume | `redis` |
+|`persistence.persistentVolumeClaim.redis.accessMode`     | The access mode of the volume | `ReadWriteOnce` |
+|`persistence.persistentVolumeClaim.redis.size`     | The size of the volume | `1Gi` |
+|`persistence.imageChartStorage.type`     | The type of storage for images and charts: `filesystem`, `azure`, `gcs`, `s3`, `swift` or `oss`. The type must be `filesystem` if you want to use persistent volumes for registry and chartmuseum. Refer to the [guide](https://github.com/docker/distribution/blob/master/docs/configuration.md#storage) for more information about the detail | `filesystem` |
+| |
+| `externalURL` | The external URL for Harbor core service | `https://core.harbor.domain` |
 | `imagePullPolicy` | The image pull policy | `IfNotPresent` |
-| **Ingress** |
-| `ingress.enabled` | Enable ingress objects | `true` |
-| `ingress.hosts.core` | The host of Harbor core service in ingress rule | `core.harbor.domain` |
-| `ingress.hosts.notary` | The host of Harbor notary service in ingress rule | `notary.harbor.domain` |
-| `ingress.annotations` | The annotations used in ingress | `true` |
-| `ingress.tls.enabled` | Enable TLS | `true` |
-| `ingress.tls.secretName` | Fill the secretName if you want to use the certificate of yourself when Harbor serves with HTTPS. A certificate will be generated automatically by the chart if leave it empty |
-| `ingress.tls.notarySecretName` | Fill the notarySecretName if you want to use the certificate of yourself when Notary serves with HTTPS. if left empty, it uses the `ingress.tls.secretName` value |
+| `logLevel` | The log level | `debug` |
+| `harborAdminPassword`  | The initial password of Harbor admin. Change it from portal after launching Harbor | `Harbor12345` |
+| `secretkey` | The key used for encryption. Must be a string of 16 chars | `not-a-secure-key` |
+| **Nginx** |
```

Comment:
Shall we document if the ingress is chosen in `expose` chunk the Nginx will not be created?